presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraIntegrationSmokeTest.java

```diff
@@ -114,6 +116,12 @@ protected boolean canDropSchema()
         return false;
     }
 
+    @Override
+    protected TestTable createTableWithDefaultColumns()
+    {
+        throw new SkipException("requirement not met");
```

Comment:
What kind of a requirement is not met? Please be more specific.

(same in other places)